cmd/orgs.go

```diff
@@ -215,6 +233,148 @@ func orgsRemove(ctx *cli.Context) error {
 	return nil
 }
 
+func orgsMembersListCmd(ctx *cli.Context) error {
+
+	// Use "member" team name to bypass specific teams
+	teamName := "member"
+
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	client := api.NewClient(cfg)
+	c := context.Background()
+
+	var getMembers sync.WaitGroup
+	getMembers.Add(2)
+
+	var org *envelope.Org
+	var orgs []envelope.Org
+	var team envelope.Team
+	var teams []envelope.Team
+	var memberships []envelope.Membership
+	var oErr, tErr, mErr, sErr error
+
+	// Retrieve the org name supplied via the --org flag.
+	// This flag is optional. If none was supplied, then
+	// orgFlagArgument will be set to "". In this case,
+	// prompt the user to select an org.
+	orgFlagArgument := ctx.String("org")
+
+	if orgFlagArgument == "" {
+		// Retrieve list of available orgs
+		orgs, oErr = client.Orgs.List(c)
+		if oErr != nil {
+			return oErr
+		}
+
+		idx, _, oErr := SelectOrgPrompt(orgs)
+		if oErr != nil {
+			return oErr
+		}
+
+		if idx == promptui.SelectedAdd {
```

Comment:
One thing we should think from a ux flow is whether or not we should allow you to create an org through the `torus orgs members` list. It seems disjointed that a user would ever want to create an org when they're listing members as the result would only include themselves.

On the other hand it fits into the way the rest of the product works.

Reply:
Agreed, this is my mistake. Looked too quickly at SelectCreateOrg which calls SelectOrgPrompt, thought SelectOrgPrompt presented the orgs without allowing creation of a new org.  Could make a new prmpt using Select instead of SelectWithAdd to disallow creation of a new org?

Reply:
Hmm, I thought a little bit more about this and while it's surprising its definitely not surprising given the way the rest of the CLI works. So I could go either way, but adding a new one for just selecting makes a lot of sense, or perhaps we open an issue and tackle it later.